src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -81,6 +81,11 @@
           text: az storage container create -n MyStorageContainer --fail-on-exist
 """
 
+helps['storage account create'] = """
```

Comment:
Before, the help text was defined in the docstring of the custom method.
I removed that docstring and added this to the help file instead.